Framework/DataHandling/inc/MantidDataHandling/LoadILLDiffraction.h

```diff
@@ -96,9 +96,10 @@ class MANTID_DATAHANDLING_DLL LoadILLDiffraction
   void resolveInstrument();
   void resolveScanType();
 
-  size_t m_sizeDim1;            ///< size of dim1, either tubes or detectors
-  size_t m_sizeDim2;            ///< size of dim2, used for pixels within tubes
-  size_t m_numberDetectorsRead; ///< number of cells read from file
+  size_t m_sizeDim1; ///< size of dim1, number of pixels in a tube (D2B) or the
+                     /// whole detector (D20)
+  size_t m_sizeDim2; ///< size of dim2, number of tubes (1 for D20!)
```

Comment:
I think this is back to front, `m_sizeDim1` should be number of tubes and `m_sizeDim2` number of pixels for D2B.